connext_cmake_module/CMakeLists.txt

```diff
@@ -1,9 +1,21 @@
 cmake_minimum_required(VERSION 2.8.3)
 
-project(connext_cmake_module NONE)
+project(connext_cmake_module)
```

Comment:
I thought `enable_language` would be the way to go but the docs suggest that it doesn't set the desired variables. Just adding a comment here why we can't use `NONE` should be good for now.